pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -2441,8 +2441,12 @@ type Image struct {
 	RepoDigests []string `protobuf:"bytes,3,rep,name=repo_digests,json=repoDigests" json:"repo_digests,omitempty"`
 	// Size of the image in bytes.
 	Size_ *uint64 `protobuf:"varint,4,opt,name=size" json:"size,omitempty"`
-	// User that will run the command(s).
-	User             *string `protobuf:"bytes,5,opt,name=user" json:"user,omitempty"`
+	// UID that will run the command(s). It is used by default if no user is
```

Comment:
UID to use to run the command(s). This is used as a default if no user is specified when creating the container.